packages/favorites-dialog/src/actions/index.js

```diff
@@ -126,11 +128,20 @@ export const fetchData = () => {
         }
 
         if (state.filtering.visTypeValue) {
-            if (state.filtering.visTypeValue !== 'all') {
-                favoriteModel = favoriteModel
-                    .filter()
-                    .on('type')
-                    .equals(state.filtering.visTypeValue);
+            switch (state.filtering.visTypeValue) {
+                case 'all':
+                    break;
+                case CHART:
+                    favoriteModel = favoriteModel
+                        .filter()
+                        .on('type')
+                        .notEqual(PIVOT_TABLE);
+                    break
+                default:
```

Comment:
Maybe not the prettiest logic to use whatever was provided to the switch case `(default)` as a filter as the fallback, but I guess only valid input will be passed in since it's based on the dropdown, not user input.

Reply:
But if you want to be safe you could check if the input is present in `visTypeMap` though...